.gitignore

```diff
@@ -4,7 +4,6 @@ shpc.db
 singularity_hpc.egg-info/
 env
 .env
-env
```

Comment:
The instructions show creating an env like this so it doesn’t hurt to have (and that’s my preference).

Reply:
Removing this line because `env` is already listed two lines above. Unless I'm misreading it ?

Reply:
Rofl. No I’m just blind don’t mind me 🤓